topdown/http.go

```diff
@@ -1331,6 +1341,10 @@ func (c *intraQueryCache) ExecuteHTTPRequest() (*http.Response, error) {
 	return executeHTTPRequest(httpReq, httpClient)
 }
 
+func (c *intraQueryCache) LockCacheKey() func() {
+	return nil
+}
```

Comment:
```suggestion
func (*intraQueryCache) LockCacheKey() func() {
	return func() {}
}
```

this way, we could just call it, without ever wondering about `!= nil`. But that's nitpicking.

Reply:
Done.